extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -70,6 +75,22 @@ export class AnchorAdStrategy {
       .includes(OPT_IN_STATUS_ANCHOR_ADS);
   }
 
+  /**
+   * Add custom amp-analytics emement
+   * @param {HTMLElement} adElement
+   */
+  addCustomAmpAnalytics_(adElement) {
+    if (this.customAnalytics_) {
+      insertAnalyticsElement(
+        adElement,
+        this.customAnalytics_,
+        false,
+        false,
+        true
```

Comment:
Please annotate bool args.

```suggestion
        /* config */ false,
        /* loadAnalytics */ false,
        /* disableImmediate */ true
```